docs/corpora_schema.md

```diff
@@ -34,100 +35,110 @@ schema. If it does not, it will reject the dataset with an appropriate error mes
+
+| Assay                                 | "raw" requirements                                                                                                                            | "raw" Required? | Final ("X") Requirements                                                                           | Final ("X") Required? | Other Layers |
+|---------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|-----------------|----------------------------------------------------------------------------------------------------|-----------------------|--------------|
+| scRNA-seq (UMI, e.g. 10x v3)          | Values MUST be de-duplicated molecule counts.                                                                                                 | REQUIRED        |                                                                                                    | STRONGLY RECOMMENDED  | OPTIONAL     |
+| scRNA-seq (non-UMI, e.g. SS2)         | Values MUST be one of read counts (e.g. FeatureCounts) or  estimated fragments (e.g. output of RSEM).                                         | REQUIRED        |                                                                                                    | STRONGLY RECOMMENDED  | OPTIONAL     |
+| Accessibility (e.g. ATAC-seq, mC-seq) |                                                                                                                                               | NOT REQUIRED    | Values MUST correspond to gene features (e.g. HGNC symbols if human)                               | REQUIRED              | OPTIONAL     |
 
 ### Integration Metadata
 
-To support data integration, each cell must have the following metadata:
+#### Cell Metadata
+
+To support data integration, each cell MUST have the following metadata:
 
 **Field name**|**Constraints**
 :--|:--
-tissue|string (see note below regarding cell culture and organoids)
+tissue|string. This field SHOULD be appended with " (cell culture)" or " (organoid)" if appropriate.
 assay|string
 disease|string
 cell\_type|string
 sex|"male", "female", "mixed", "unknown", or "other"
 ethnicity|string, "na" if non-human, "unknown" if not available
 development\_stage|string, "unknown" if not available
 
-The `tissue` field must be appended with " (cell culture)" or " (organoid)" if appropriate. Also, if the source of cells is cell culture or organiod, the
-`cell_type` field can be left empty.
-In addition to these free text fields (except sex), each cell must also have ontology values:
+In addition to these free text fields (except sex), each cell MUST also have ontology annotations:
+
+Ontology terms MUST use [OBO-format ID](http://www.obofoundry.org/id-policy.html), meaning they are a CURIE
+where the prefix identifies the ontology. For example `EFO:0000001` is a term in the `EFO` ontology.
 
```

Comment:
Minor editorial. Both there's a hanging `:` and the sentences can be merged? 

Also - don't you want the label to match the ontology? (["blood" versus "Blood" for example](https://chanzuckerbergteam.slack.com/archives/C018B64J3HN/p1617144334028900)) These are really not "free text".

Reply:
Great catch. Addressed in #52